composer.json

```diff
@@ -8,28 +8,28 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^8.0.2",
+        "php": "^8.1.0",
         "anlutro/l4-settings": "^1.1.1",
         "doctrine/dbal": "^3.3.6",
-        "graham-campbell/markdown": "^14.0",
+        "graham-campbell/markdown": "^15.0",
         "guzzlehttp/guzzle": "^7.4.2",
         "hisorange/browser-detect": "^4.5",
-        "laravel/framework": "^9.0",
-        "laravel/tinker": "^2.0",
-        "laravel/ui": "^3.2",
+        "laravel/framework": "^10.0",
+        "laravel/tinker": "^2.8",
+        "laravel/ui": "^4.0",
```

Comment:
iirc, I don't think we use laravel/ui at all, or..?

Reply:
Oh, that's interesting! Well, for scope's sake I recommend removing that in a separate PR -- even though it might be a very simple change.

Reply:
WDYT?

Reply:
Agree, a own PR. Could be just another rabbit hole 🐰